obal/__init__.py

```diff
@@ -321,7 +321,11 @@ def main(cliargs=None):  # pylint: disable=R0914
     from ansible.cli.playbook import PlaybookCLI
 
     ansible_args = generate_ansible_args(inventory_path, args, parser.obal_arguments)
-    ansible_playbook = (["ansible-playbook"] + ansible_args)
+    ansible_playbook = (
+        ["ansible-playbook"]
+        + ["-e", "ansible_python_interpreter='/usr/bin/env python3'"]
```

Comment:
Does this degrade gracefully for systems that only have python2?

Reply:
No. Ansible 2.8 will have autodetection for this so it will no longer be needed, but this breaks EL7 support.

Reply:
I think it's about time that obal decides to support either py2 or py3. @ehelms, @ekohl thoughts?

Reply:
:-1: it means we can no longer run obal on EL7 machine. In the Foreman infra we currently don't have any Fedora slaves and I'd like to avoid installing them just to run obal. However, I still want to use obal on my Fedora machine.

Reply:
i removed this section where `ansible_python_interpreter` was being set.